### PR TITLE
Dump installer logs on failure

### PR DESCRIPTION
**Pull request by jherico**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22782/HighFidelity-Beta-PR16559-4f13fc8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22782/HighFidelity-Beta-Interface-PR16559-4f13fc8.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22782/HighFidelity-Beta-PR16559-4f13fc8.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22782/HighFidelity-Quest-Beta-PR16559-4f13fc8.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22782/HighFidelity-Beta-PR16559-4f13fc8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22782/HighFidelity-Beta-Interface-PR16559-4f13fc8.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22786/HighFidelity-Beta-PR16559-86dd2e2.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22786/HighFidelity-Quest-Beta-PR16559-86dd2e2.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22786/HighFidelity-Beta-PR16559-86dd2e2.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22786/HighFidelity-Beta-Interface-PR16559-86dd2e2.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22789/HighFidelity-Beta-PR16559-ca729cb.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22789/HighFidelity-Quest-Beta-PR16559-ca729cb.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22789/HighFidelity-Beta-PR16559-ca729cb.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22789/HighFidelity-Beta-Interface-PR16559-ca729cb.dmg

File: .github/workflows/pr_build.yml [hunk 1]
```diff
@@ -2,87 +2,95 @@ name: Pull Request CI Build
...
 jobs:
-  build_full_mac:
-    runs-on: macOS-latest
-    if: github.event.action != 'labeled' || github.event.label.name == 'rebuild-mac' || github.event.label.name == 'rebuild'
+  build:
+    strategy:
+        matrix:
+          os: [windows-latest, macOS-latest]
+          build_type: [full]
+    runs-on: ${{matrix.os}}
```
Comment (SimonWalton-HiFi) [inline .github/workflows/pr_build.yml]: Any reason not to have `fail-fast: false` here, so other builds continue when one fails?

Comment (jherico) [reply .github/workflows/pr_build.yml]: good call

**Pull request merged**